ngraph/frontend/onnx_import/include/onnx_import/utils/onnx_internal.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
@postrational @tomdol 
We've discussed offline that this internal header should be in `src` folder.
Unfortunately, it is not visible for `onnx_editor` in such a case (in the current cmake configuration).
As a result I moved it to `utils` subfolder.